rs/replay/src/player.rs

```diff
@@ -764,32 +764,41 @@ impl Player {
                 )
             }
         };
+
+        let context_time = time;
+        let extra_msgs = extra(self, context_time);
+        if extra_msgs.is_empty() {
+            return (context_time, None);
+        }
+
+        let extra_ingresses = if !extra_msgs.is_empty() {
+            println!("extra_batch created with new ingress");
+            extra_msgs
+                .iter()
+                .map(|fm| fm.ingress.clone())
+                .collect::<Vec<_>>()
+        } else {
+            vec![]
+        };
```

Comment:
```suggestion
        let extra_ingresses = extra_msgs
            .iter()
            .map(|fm| fm.ingress.clone())
            .collect::<Vec<_>>();
```
From the previous check, `extra_msgs` is non empty, right..?

And I guess I would put the `println!("extra_batch created with new ingress");` after actually having created the `extra_batch`?

Reply:
Good point! Changed.